src/main/java/org/jenkinsci/test/acceptance/plugins/git/GitScm.java

```diff
@@ -103,6 +111,16 @@ public GitScm calculateChangelog(String remote, String branch) {
     }
 
     /**
+     * Add behaviour "Use commit author in changelog"
+     *
+     * @return this, to allow function chaining
+     */
+    public GitScm commitAuthorInChangelog() {
+        CommitAuthorInChangelog behaviour = addBehaviour(CommitAuthorInChangelog.class);
```

Comment:
No need to store this in a variable then?

Reply:
@varyvol you're right!